src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/Helpers/VariableLengthIntegerHelper.cs

```diff
@@ -32,9 +32,9 @@ internal static class VariableLengthIntegerHelper
         private const ulong EightByteLengthMask = 0xC000000000000000;
 
         public const uint OneByteLimit = (1U << 6) - 1;
-        private const uint TwoByteLimit = (1U << 16) - 1;
-        private const uint FourByteLimit = (1U << 30) - 1;
-        private const long EightByteLimit = (1L << 62) - 1;
+        public const uint TwoByteLimit = (1U << 16) - 1;
```

Comment:
Nit: Consider adding a comment that the consts are used by an aspnetcore API. Avoid them accidently being made private in the future